README.rst

```diff
@@ -33,7 +33,7 @@ a two-qubit circuit.
 
     circuit = Circuit(2)
 
-    with circuit.context as q:
+    with circuit.context as (q, c):
```

Comment:
API idea: introduce a `namedtuple` or `tuple` subclass like `Registers` (or `Context`?) and return that instead of a plain tuple from `CircuitContext.__enter__`.

That way users can be more or less verbose:
```
with circuit.context as (q, c):
    ops.Hadamard(q[1])
    ...
```
or
```
with circuit.context as registers:
    ops.Hadamard(registers.q[1])
    ...
```